Library/ENV/4.3/cc

```diff
@@ -276,6 +281,11 @@ class Cmd
     %W[#{sysroot}/usr/lib /usr/local/lib]
   end
 
+  def keg_path
+    return nil if formula.nil?
+    "#{cellar}/#{formula}/#{formula_version}"
```

Comment:
The fix I'm doing for ignoring the revisions should address this, too: it constructs the whole prefix path using the `Formula` log in superenv in the parent `brew` process and just passes the whole path, instead of reconstructing it in the `cc` wrapper script. https://github.com/apjanke/brew/commit/bc51fd5944135cea74c91d7d7a5eff97b1f07f27

And I think in this case, the nil vs empty string distinction doesn't matter because both are false, and it does an `if formula_prefix` check before using it.